databricks/koalas/series.py

```diff
@@ -3223,6 +3223,90 @@ def copy(self) -> 'Series':
         """
         return _col(DataFrame(self._internal.copy()))
 
+    def truncate(self, before=None, after=None, copy=True):
+        """
+        Truncates a sorted Series before and/or after some particular index value.
+        Series should have sorted index.
+
+        Parameters
+        ----------
+        before : string, int
+            Truncate all rows before this index value
+        after : string, int
+            Truncate all rows after this index value
+        copy : boolean, default is True,
+            return a copy of the truncated section
+
+        Returns
+        -------
+        truncated : Series
+
+        Examples
+        --------
+
+
+        A Series has index that sorted integers.
+
+        >>> s = ks.Series([10, 20, 30, 40, 50, 60, 70],
+        ...               index=[1, 2, 3, 4, 5, 6, 7])
+        >>> s
+        1    10
+        2    20
+        3    30
+        4    40
+        5    50
+        6    60
+        7    70
+        Name: 0, dtype: int64
+
+        >>> s.truncate(2, 5)
+        2    20
+        3    30
+        4    40
+        5    50
+        Name: 0, dtype: int64
+
+        A Series has index that sorted strings.
+
+        >>> s = ks.Series([10, 20, 30, 40, 50, 60, 70],
+        ...               index=['a', 'b', 'c', 'd', 'e', 'f', 'g'])
+        >>> s
+        a    10
+        b    20
+        c    30
+        d    40
+        e    50
+        f    60
+        g    70
+        Name: 0, dtype: int64
+
+        >>> s.truncate('b', 'e')
+        b    20
+        c    30
+        d    40
+        e    50
+        Name: 0, dtype: int64
+        """
+        indexes = self.index.to_pandas()
```

Comment:
@itholic, this will collect everything in index into driver's memory. We should avoid this.

Reply:
Koalas index implements `is_monotonic_increasing` and `is_monotonic_decreasing`. The problem is that this is an expensive operation as warned in their documentations as it leads all data into single node.

Can you add a note that this API will be expensive (see `is_monotonic_increasing` as an example). We even might have to explicitly don't implement. cc @ueshin

Reply:
@HyukjinKwon Thanks for commenting!

yeah you're right. i used `is_monotonic_increasing` of pandas' because there was a bug for koalas'. (so i fixed in #930 )

now we can use koalas', but there still have an issue about cost as you said.